src/ocl/convolutionocl.cpp

```diff
@@ -2323,7 +2323,7 @@ void ConvolutionDescriptor::FindConvBwdDataAlgorithm(Handle& handle,
 
     std::vector<PerfField> perf_db;
 
-    const miopen::FindMode fm;
+    const miopen::FindMode fm(problem);
```

Comment:
Lingering naming convention issue with using `problem` and `ctx` depending on the convolution direction.

Reply:
No, this is really a `problem`, an instance of the `ProblemDescription` class. The temporary instance of `ConvolutionContext` is implicitly created from it just prior the invocation of `FindMode()`.